docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -29,6 +29,18 @@ and then read metadata from each data file.
 Since Iceberg stores the paths to data files in the metadata files, it
 only consults the underlying file system for files that must be read.
 
+Requirements
+------------
+
+To use Iceberg, you need:
+
+* Iceberg table format compatible with Apache Iceberg format version 1.
```

Comment:
This may seem a bit confusing to someone starting with Iceberg. I guess we want to say that Trino is able to read Iceberg tables that have metadata stored in a format that complies with Spec version 1. But this isn't something that user "needs". Also, they can still create Iceberg tables through Trino (although not performant atm). May be we can put this in overview or reword? @mosabua @hashhar

Reply:
This is the requirements section and as such the current setup and wording is appropriate. We will add more info and context about how it works and hangs together in future updates.